languages/scala/reference/README.md

```diff
@@ -0,0 +1,211 @@
+# Scala reference
+
+## Concepts
+
+The Scala concept exercises are based on concepts. The list below contains the concepts that have been identified for the Scala language.
+
+### Language-unique
+
+- Implicits
+- path-dependent-path
```

Comment:
It should be Path-dependent `types` (not path-dependent-path)

Reply:
@RafaelBa Did you know that you can make inline suggestions: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request

This makes it very easy for the PR submitter to apply changed. I'll give an example below.

```suggestion
- Path-dependent types
```

Reply:
Oh neat! I'll keep that in mind, thanks.